.github/workflows/release-pipeline.yml

```
@@ -356,6 +356,18 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+
+      - name: Verify bifrost-http release
+        id: verify
+        continue-on-error: true
+        env:
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:2:25: Double quote to prevent globbing and word splitting

(shellcheck)